src/extension.ts

```diff
@@ -1081,31 +1083,36 @@ async function isBashOnPod(podName : string): Promise<boolean> {
 
 async function syncKubernetes() : Promise<void> {
```

Comment:
From the command name `Kubernetes: Sync`, i totally don't understand what it did. Can we change it to `Git-Checkout` or `Checkout` directly?

Reply:
I agree it's a pretty cryptic name.  However, I think of checkout as an implementation detail (admittedly a pretty darned blatant one!).  I've proposed `Sync Working Copy to Cluster`, and added some docs.  Thoughts?

Reply:
Yeah, the new name is better.